actors/market/src/lib.rs

```diff
@@ -77,6 +77,7 @@ pub enum Method {
     AddBalanceExported = frc42_dispatch::method_hash!("AddBalance"),
     WithdrawBalanceExported = frc42_dispatch::method_hash!("WithdrawBalance"),
     GetBalanceExported = frc42_dispatch::method_hash!("GetBalance"),
+    GetDealActivationExported = frc42_dispatch::method_hash!("GetDealActivation"),
```

Comment:
Not sure that this is the best name given we're including termination info, but :woman_shrugging:. I think you're right to avoid calling it something like `GetDealState`.